src/mpcCoreKit.ts

```diff
@@ -310,6 +314,17 @@ export class Web3AuthMPCCoreKit implements ICoreKit {
     }
 
     // if not redirect flow or session rehydration, ask for factor key to login
+
+    // dont wait for wasm to be loaded, we can reload it during signing if not loaded
+    this._tssLib
```

Comment:
I think it might be better to wait here.
Loading should be relatively fast, because the WASM file is prepackaged.

Otherwise, we might get some unexpected errors.

Reply:
my reasoning it wasm is only used while signing , which is done after login which will be done. So we can just return from init function without waiting and i m checking later if wasm is not loaded thn i m loading it during signing

Reply:
also if you think loading wasm is quite faster and doesnt require loading it lazily thn i can just remove it.

Reply:
measured it. loading takes 30ms on my machine. up to you.